librustzcash/src/rustzcash.rs

```diff
@@ -206,30 +215,36 @@ fn init_zksnark_params(
         .to_str()
         .expect("hash should be a valid string");
 
-    let sprout_hash = unsafe { CStr::from_ptr(sprout_hash) }
-        .to_str()
-        .expect("hash should be a valid string");
+    let sprout_hash_option = if sprout_path.is_none() {
```

Comment:
```suggestion
    let sprout_hash = if sprout_path.is_none() {
```

It would also be good to move the hashes internally, but we can split that out from #85 into a separate issue.

Reply:
Opened #106 for moving the hashes.